presentation/frontend/src/presentation-frontend/PresentationManager.ts

```diff
@@ -278,7 +262,14 @@ export class PresentationManager implements IDisposable {
    * Function that is called when a new IModelConnection is used to retrieve data.
    * @internal
    */
-  public async onNewiModelConnection(_: IModelConnection) {}
+  public startiModelInitialization(_: IModelConnection) {}
```

Comment:
Let's rename `startiModelInitialization` and `ensureiModelInitialized` to `startIModelInitialization` and `ensureIModelInitialized` to make them more readable.

Reply:
Ok, done